ignite/services/network/network.go

```diff
@@ -4,20 +4,39 @@ import (
 	"context"
 	"strconv"
 
-	"github.com/pkg/errors"
-
+	"github.com/cosmos/cosmos-sdk/client"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ignite-hq/cli/ignite/pkg/cosmosaccount"
 	"github.com/ignite-hq/cli/ignite/pkg/cosmosclient"
 	"github.com/ignite-hq/cli/ignite/pkg/events"
+	"github.com/pkg/errors"
+	campaigntypes "github.com/tendermint/spn/x/campaign/types"
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+	profiletypes "github.com/tendermint/spn/x/profile/types"
+	rewardtypes "github.com/tendermint/spn/x/reward/types"
 )
 
+//go:generate mockery --name CosmosClient --case underscore
```

Comment:
Let's add `mockery` as a dependency in `ignite/internal/tools/tools.go` to lock it's version.

https://play-with-go.dev/tools-as-dependencies_go115_en/